osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/ConfigurationManagementServiceTest.java

```diff
@@ -0,0 +1,290 @@
+        final long scheduleTime = 1;
+        final String messageType = "none";
+        final int messagePriority = 1;
+
+        final Organisation testOrganisation = new Organisation();
+
+        final RelayMap relayMap = new RelayMap(1, 1, RelayType.LIGHT, "1");
+        final ArrayList<RelayMap> relayMapList = new ArrayList<>();
+        relayMapList.add(relayMap);
+
+        when(this.organisationDomainService.searchOrganisation(any(String.class))).thenReturn(testOrganisation);
+        when(this.deviceDomainService.searchActiveDevice(any(), any())).thenReturn(this.device);
+        when(this.configuration.getRelayConfiguration()).thenReturn(new RelayConfiguration(relayMapList));
+        when(this.ssldRepository.findById(any())).thenReturn(java.util.Optional.of(this.ssld));
+        when(this.domainCoreMapper.map(any(), any())).thenReturn(this.configurationDto);
+        doNothing().when(this.ssld).updateOutputSettings(any());
+        when(this.device.getIpAddress()).thenReturn("333.333.1.22");
+
+        this.configurationManagementService.setConfiguration(this.ids, this.configuration, scheduleTime,
+                messageType, messagePriority);
+
+        verify(this.osgpCoreRequestMessageSender).sendWithScheduledTime(this.requestMessageArgumentCaptor.capture(),
+                this.stringArgumentCaptorOne.capture(), this.integerArgumentCaptor.capture(),
+                this.stringArgumentCaptorTwo.capture(), this.longArgumentCaptor.capture());
+
+        assertThat(this.requestMessageArgumentCaptor.getValue().getCorrelationUid()).isEqualTo("c");
+        assertThat(this.requestMessageArgumentCaptor.getValue().getOrganisationIdentification()).isEqualTo("a");
+        assertThat(this.requestMessageArgumentCaptor.getValue().getDeviceIdentification()).isEqualTo("b");
+        assertThat(this.stringArgumentCaptorOne.getValue()).isEqualTo(messageType);
+        assertThat(this.integerArgumentCaptor.getValue()).isEqualTo(1);
+        assertThat(this.stringArgumentCaptorTwo.getValue()).isEqualTo("333.333.1.22");
+        assertThat(this.longArgumentCaptor.getValue()).isEqualTo(1);
+    }
+    @Test
+    public void testTrySetConfigurationWithNoConfiguration() throws Exception {
+        final long scheduleTime = 1;
+        final String messageType = "none";
+        final int messagePriority = 1;
+
+        final Organisation testOrganisation = new Organisation();
+        when(this.organisationDomainService.searchOrganisation(any(String.class))).thenReturn(testOrganisation);
+        when(this.deviceDomainService.searchActiveDevice(any(), any())).thenReturn(new Device());
+        when(this.organisationDomainService.searchOrganisation(any(String.class))).thenReturn(testOrganisation);
+
+        this.configurationManagementService.setConfiguration(this.ids, null, scheduleTime, messageType, messagePriority);
+
+        assertThat(this.outContent.toString().contains("Configuration is empty, skip sending a request to device")).isTrue();
```

Comment:
replace this line with
assertThat(this.outContent.toString()).contains("Configuration is empty, skip sending a request to device");